src/config/http.rst

```diff
@@ -245,6 +232,36 @@ HTTP Server Options
            upgrade, it is advisable to review the usage of these configuration
            settings.
 
+.. config:section:: httpd :: HTTP Server Options
+
```

Comment:
Would a section-based `.. versionchanged:: 3.2 ` would work indicating the list of options which moved

```
  .. config:section:: httpd :: HTTP Server Options
  .. versionchanged:: 3.2 These options were moved to chttpd section: `allow_jsonp`, `max_http_request_size`,  `x_forwarded_ssl`, ....
```

Those might not render as nicely so then we could skip it probably

Reply:
Added, thanks for the review, especially for the list.